azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallCompositeExtentions.kt

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling
+
+import android.content.Context
+import com.azure.android.communication.ui.calling.data.CallHistoryRepositoryImpl
+import com.azure.android.communication.ui.calling.presentation.manager.DebugInfoManager
+import com.azure.android.communication.ui.calling.presentation.manager.DebugInfoManagerImpl
+
+internal fun createDebugInfoManager(context: Context): DebugInfoManager {
+    return DebugInfoManagerImpl(CallHistoryRepositoryImpl(context))
```

Comment:
use context.applicationContext here if possible

Reply:
Good idea, moved context.applicationContext higher in call stack